partials/forms/checkbox.html

```diff
@@ -1,6 +1,6 @@
 <div id="{{key}}-group" class="form-group{{#compound}} form-group-compound{{/compound}}{{#error}} validation-error{{/error}}">
     {{#error}}<p class="error-message" aria-hidden="true">{{error.message}}</p>{{/error}}
-    <label for="{{key}}" class="block-label{{#invalid}} invalid-input{{/invalid}}">
+    <label for="{{key}}" class="{{#className}} {{className}}{{/className}}{{#invalid}}invalid-input{{/invalid}}">
```

Comment:
The whitespace here will result in the `{{className}}` and `invalid-input` being run together. 

`{{#className}} {{className}}{{/className}}` should be `{{#className}}{{className}} {{/className}}`